README.md

```diff
@@ -11,12 +11,22 @@ This plugin will download metadata about Youtube videos using a Youtube API key.
 1. Clone or download this repository.
 1. Ensure you have .NET Core SDK setup and installed.
 1. Build plugin with following command.
-    ```sh
+    ```
     dotnet publish --configuration Release --output bin
     ```
-1. Create folder named `YoutubeMetadata` in the `plugin` directory within `data` directory.
-1. Place the resulting file from step 3 in the `plugins/YoutubeMetadata` folder created in step 4.
-1. If performed correctly you will see the plugin in the dashboard plugin page.
+1. Create folder named `YoutubeMetadata` in the `plugins` directory inside your Jellyfin data
+   directory, usually `/var/lib/jellyfin`. 
+1. Place the resulting files from step 3 in the `plugins/YoutubeMetadata` folder created in step 4.
+    ```
+    $ chmod +x bin/*.dll
```

Comment:
Do dll files need to have the x bit enabled? I don't believe I did this for mine.

Reply:
I do not know. I have just noticed every `ddl` file from dotnet applications that run on linux are `+x`. My apologies

Reply:
I don't believe they need to be +x. Mine run fine and I do not have execution set. Remove that and I can merge this.